special-pages/pages/new-tab/app/omnibar/integration-tests/omnibar.spec.js

```diff
@@ -49,21 +49,26 @@ test.describe('omnibar widget', () => {
         });
     });
 
-    test('AI chat button in search form submits chat message', async ({ page }, workerInfo) => {
+    test('AI chat form shift+enter creates new line', async ({ page }, workerInfo) => {
         const ntp = NewtabPage.create(page, workerInfo);
         const omnibar = new OmnibarPage(ntp);
         await ntp.reducedMotion();
 
         await ntp.openPage({ additional: { omnibar: true } });
         await omnibar.ready();
 
-        await omnibar.searchInput().fill('pizza');
-        await omnibar.aiChatButton().click();
+        await omnibar.aiTab().click();
+        await omnibar.expectMode('ai');
 
-        await omnibar.expectMethodCalledWith('omnibar_submitChat', {
-            chat: 'pizza',
-            target: 'same-tab',
-        });
+        await omnibar.chatInput().fill('first line');
+        await omnibar.chatInput().press('Shift+Enter');
+        await omnibar.chatInput().pressSequentially('second line');
+
+        // Check that the textarea contains both lines with a newline
+        await expect(omnibar.chatInput()).toHaveValue('first line\nsecond line');
+
+        // Verify that the form was not submitted (no method call should have been made)
+        await omnibar.expectMethodNotCalled('omnibar_submitChat');
```

Comment:
Can we add a control in the same test, lots of ways this can pass otherwise

Reply:
What do you mean, sorry? We have `test('AI chat form submission'` above which checks pressing <kbd>Enter</kbd> without holding <kbd>Shift</kbd>.